docs/configuration/scripts.mdx

```diff
@@ -77,7 +77,7 @@ scripts:
       concurrency: 1
 ```
 
-See the [`select-package`](scripts#scriptsselect-package) option for filtering the packages to execute
+See the [`select-package`](configuration/scripts#scriptsselect-package) option for filtering the packages to execute
```

Comment:
Almost there 😉. The URL needs to start with `/`.

Reply:
I don't think this is necessary, because `scripts#scriptsselect-package` gives `https://melos.invertase.dev/#scriptsselect-package`

Reply:
I just tried this PR in a [local preview](https://use.docs.page/previews#local-preview-mode), and it seems URL have to have a path and that path must be absolute.